tests/examples/opinions/united_states/bia_example.compare.json

```diff
@@ -0,0 +1,578 @@
+[
+  {
+    "case_dates": "2021-07-01",
+    "case_names": "VUCETIC",
+    "download_urls": "/eoir/file/1381766/download",
```

Comment:
I think CL will struggle to download this URL. :)

Want to fix this and add a tweak to the tests to make sure that every url starts with `http`?

Reply:
You know - I actually don't think this is correct.

Reply:
Take a look at the Alaska for example.  It also has the abbreviated urls in the testing json file but I double checked and its not because we don't append the full url in the scrape.  I'm not sure why some cases have full urls and others don't but this actually should work and bring back the full url when using juriscraper.

Reply:
Hmmmmmmmm, that's weird. I'm surprised it'd work either way, but can you dig deeper to figure out which is better and then normalize our tests on one or the other (and add a test to make sure we don't regress)?

Reply:
Yep